hax/hax/server.py

```diff
@@ -262,7 +262,8 @@ def fn():
                 proc_fid = Fid.parse(item['Key'].split('/')[1])
                 proc_state = proc_status['state']
                 proc_type = proc_status['type']
-                if (proc_type == 'M0_CONF_HA_PROCESS_M0D' and
+                if (proc_type in ('M0_CONF_HA_PROCESS_M0D',
```

Comment:
Please change this condition as follows,
```
if (proc_type != 'M0_CONF_HA_PROCESS_M0MKFS' and
    proc_state in ('M0_CONF_HA_PROCESS_STARTED',
                           'M0_CONF_HA_PROCESS_STOPPED')):
```
and please try deployment.

Reply:
means for `proc_type` we are expecting for `M0_CONF_HA_PROCESS_KERNEL` and `M0_CONF_HA_PROCESS_OTHER` too with `M0_CONF_HA_PROCESS_M0D` and `M0_CONF_HA_PROCESS_HA`

Reply:
KERNEL is obsolete now, but yes for all others except MKFS.